### PR TITLE
Remove foo, bar... in tests

### LINKED ISSUE #3213: Do not use "Foo" or "Bar" as name in tests
These names should not be used in tests as they generates some issues internally.

### PR DESCRIPTION
**Pull request by Evangelink**
Fix #3213

### CODE REVIEW

**Comment by mavasani** [comment]
@Evangelink Tests seem to be hanging in this PR. Can you try locally?

**Comment by Evangelink** [comment]
It seems to be ok from the `Test Explorer` but not from the command line. That's quite weird because there is no functional change...

File: src/Microsoft.CodeAnalysis.Analyzers/UnitTests/InternalImplementationOnlyTests.cs [hunk 1]
```diff
@@ -517,265 +517,249 @@ public async Task Basic_VerifyISymbol()
         {
             var source = @"
 ' Causes many compile errors, because not all members are implemented.
-Class Foo 
+Class C1
     Implements Microsoft.CodeAnalysis.ISymbol
 End Class
-Class Bar
+Class C2
```
Comment (Evangelink) [inline src/Microsoft.CodeAnalysis.Analyzers/UnitTests/InternalImplementationOnlyTests.cs]: Using `SomeClass` and `SomeOtherClass` here was causing the test to hang... I couldn't find out why (I didn't spent too long debugging it).

File: src/Microsoft.CodeAnalysis.Analyzers/UnitTests/InternalImplementationOnlyTests.cs [hunk 2]
```diff
@@ -517,265 +517,249 @@ public async Task Basic_VerifyISymbol()
...
         public async Task Basic_VerifyIOperation()
         {
             var source = @"
 ' Causes many compile errors, because not all members are implemented.
-Class Foo 
+Class C1
     Implements Microsoft.CodeAnalysis.IOperation
 End Class
-Class Bar
+Class C2
```
Comment (Evangelink) [inline src/Microsoft.CodeAnalysis.Analyzers/UnitTests/InternalImplementationOnlyTests.cs]: Using `SomeClass` and `SomeOtherClass` here was causing the test to hang... I couldn't find out why (I didn't spent too long debugging it).

**Pull request merged**